src/components/form-template/form-template.component.tsx

```diff
@@ -33,7 +34,9 @@ export const FormTemplate = ({ title, required = false, children }: Props): JSX.
         <div className="d-flex align-items-center">
           <h2 className="mb-0 text-ellipsis">{title}</h2>
           {required && (
-            <span className="fdk-badge badge fdk-bg-color-warning-lightest ml-2">{localization.required}</span>
+            <Can I="view" a="Required badge">
```

Comment:
Form template ser ut som generisk bibliotek-komponent, so det er anbefalt at det tar alt input gjennom props. 
Vi allerede bryter den regel ved localisation (bruker valgt spsåk state) men det kan være en unntak.
Jeg anbefaler å ikke introdusere mer global state her, i stedet ha en ekstra enkel parameter "readonly".  `{required && !readonly && (<...>)}` Eller, alternativt, kaller kan også  bare sette required false, men det ser allerede lit hackish

Reply:
Gjør på denne måten for helptext og form-template